pkg/blkid/blkid.go

```diff
@@ -1,3 +1,6 @@
+//go:build (linux && ignore) || cgo
```

Comment:
What is this build tag used for?

Reply:
Sorry, this and the below line changed because of my local environment is go 1.17.1 . The `gofmt` cause the change. The help described this: [go help](https://pkg.go.dev/cmd/go/internal/help)

```
To keep a file from being considered for the build:

	//go:build ignore

(any other unsatisfied word will work as well, but "ignore" is conventional.)

To build a file only when using cgo, and only on Linux and OS X:

	//go:build cgo && (linux || darwin)
```

It just a mark make us can set the file `ignore` when compiling.